lib/msf/ui/console/command_dispatcher/evasion.rb

```diff
@@ -50,22 +50,32 @@ def cmd_rerun(*args)
 
   alias cmd_rexploit cmd_rerun
 
-  def cmd_exploit_tabs(str, words)
+  #
+  # Tab completion for the run command
+  #
+  def cmd_run_tabs(str, words)
```

Comment:
For consistency it might be easier to rename this back to `cmd_exploit_tabs`, and and an alias for `cmd_run_tabs` instead

```suggestion
  def cmd_exploit_tabs(str, words)
```

Reply:
Ok, will look into this.